CMakeLists.txt

```diff
@@ -102,31 +108,29 @@ if (LIBWEBSOCKETS_FOUND AND LIBWEBSOCKETS_VERSION_STRING)
 endif(LIBWEBSOCKETS_FOUND AND LIBWEBSOCKETS_VERSION_STRING)
 
 # Python
-if (PYTHON_VERSION_MAJOR STREQUAL 3)
+if (Python_VERSION_MAJOR STREQUAL 3)
     set(PY_STRING "python3")
-elseif(PYTHON_VERSION_MAJOR STREQUAL 2)
+elseif(Python_VERSION_MAJOR STREQUAL 2)
     set(PY_STRING "python")
 endif()
 
-enable_testing()
-
 # python unittest
 # For versions < 2.7 the developer needs to install the unittest2
```

Comment:
We are not doing versions < 2.7 anymore, right? This PR needs to be updated ?

Reply:
Thanks for the notice. I'll rebase the PR when I'm at a computer.

Reply:
Rebased.